data/filedefs/filetypes.javascript

```diff
@@ -3,8 +3,8 @@
 
 [keywords]
 # all items must be in one line
-primary=break case catch class const continue default delete do each else extends false finally for function get if in Infinity instanceof let NaN new null return set static super switch this throw true try typeof undefined var void while with yield prototype
-secondary=Array Boolean Date Function Math Number Object String RegExp EvalError Error RangeError ReferenceError SyntaxError TypeError URIError constructor prototype decodeURI decodeURIComponent encodeURI encodeURIComponent eval isFinite isNaN parseFloat parseInt
+primary=break case catch class const continue debugger default delete do else enum export extends false finally for function get if import in Infinity instanceof let NaN new null return set static super switch this throw true try typeof undefined var void while with yield prototype async await
```

Comment:
It's because async/await expressions [are in the draft for ES2017](https://tc39.github.io/ecmascript-asyncawait/#async-function-definitions) but [await expressions have to be inside an async function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). Many browsers already support it: Chrome, Firefox and Opera, and Babel compile it to legacy code for other platforms; so people can already program using async/await of ES7, and people actually do.
I've checked that other tools highlight the word: Atom, Chrome Dev Tools,...